docs/installing-solidity.rst

```diff
@@ -278,8 +278,8 @@ in Visual Studio 2017 Build Tools or Visual Studio 2017:
 * Windows 8.1 SDK
 * C++/CLI support
 
-.. _Visual Studio 2017: https://www.visualstudio.com/vs/
-.. _Visual Studio 2017 Build Tools: https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2017
+.. _Visual Studio 2017: https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2017-family
+.. _Visual Studio 2017 Build Tools: https://chocolatey.org/packages/visualstudio2017buildtools
```

Comment:
I can't find any official page that is not behind a login and lets you download VS 2017 Build Tools. There's [this direct link to vs_buildtools.exe](https://download.visualstudio.microsoft.com/download/pr/3e542575-929e-4297-b6c6-bef34d0ee648/639c868e1219c651793aff537a1d3b77/vs_buildtools.exe) which was suggested in [Are Visual Studio 2017 Build Tools still available for download?](https://stackoverflow.com/questions/57795314/are-visual-studio-2017-build-tools-still-available-for-download) but I don't have any way to verify if it's the right version and if it's really an installer for Build Tools so putting it blindly in the official docs is risky.

@christianparpart Does this still require VS 2017 or maybe VS 2019 would work too?

Reply:
@christianparpart said that he's working on building with VS2019 in #9476 so I'm going to leave these links unchanged there and leave it up to him to update them in that PR.